src/module/os.c

```diff
@@ -109,6 +109,20 @@ void processExit(WrenVM* vm) {
   uv_stop(getLoop());
 }
 
+// chdir_(dir)
+void processChdir(WrenVM* vm)
+{
+  wrenEnsureSlots(vm, 1);
+  char* dir = cli_strdup(wrenGetSlotString(vm, 1));
+  int err = uv_chdir(dir);
+  if (err)
+  {
+    wrenSetSlotString(vm, 0, "Cannot change directory.");
```

Comment:
It would be nice to make the error more dynamic. The [libuv api documentation](http://docs.libuv.org/en/v1.x/misc.html?highlight=chdir#c.uv_chdir) is not very illuminating about what errors could be returned.

Reply:
I assume POSIX error codes.... we've used `printf` in a single place (process handling) to add more context but in general that's bad since these errors could be caught and handled (in which case they should be silent)... but I agree in principal that merging the error code into the error string would be nice (or even returning an error object), but we'd want to do that across the board - and that's a much larger change.

I still hope that CLI and Console can ultimately share some patches back and forth so trying not to go crazy with the C module stuff yet (other than extending it minimally as needed).  Ruby has said she'll have more time for CLI in the near future, so we'll see what comes of that.